src/relayserver/RelayServer.js

```diff
@@ -270,39 +311,70 @@ class RelayServer extends EventEmitter {
       })
   }
 
+  fatal (message) {
+    console.error('FATAL: ' + message)
+    process.exit(1)
+  }
+
+  async _init () {
+    const relayHubAddress = this.relayHubContract.options.address
+    console.log('Server address', this.managerAddress)
+    const code = await this.web3.eth.getCode(relayHubAddress)
```

Comment:
This check seems redundant when you check version right after.

Reply:
it is for adding better error handling: I don't like diagnostics like "no contract at address or contract is not a RelayHub" since it doesn't really help the user.